Quick improvements to 12/13  release notes (#1651)
astronomer/docs: astro/release-notes.md

The [Cloud IDE](cloud-ide/overview.md) includes several new features which improve DAG authoring and testing:

> - Committing to GitHub and configuring a GitHub repository are now in two separate menus dedicated to each function.

Reviewer: ```suggestion
- Two separate menus are now available for GitHub commits and for configuring a GitHub repository.
```

- The default CI/CD pipeline included in the Cloud IDE project supports DAG-only deploys. Using the CI/CD pipeline to deploy DAG changes to Astro is now significantly faster.

> - The GitHub configuration menu now includes **Clone Repo** settings. You can use these settings to make files in your configured GitHub repository accessible when you run DAGs in the Cloud IDE, such as helper functions in an `include` folder.

Reviewer: ```suggestion
- The **Configure GitHub** dialog in the Cloud UI now includes a **Clone Repo** toggle. Enabling this option makes files such as helper functions in an `include` folder in your configured GitHub repository accessible when you run DAGs in the Cloud IDE.
```

- `n2-highcpu-8`
- `n2-highcpu-16`

For more information about these worker types, see [N2 machine series](https://cloud.google.com/compute/docs/general-purpose-machines#n2_machines). For a list of all worker types available on GCP, see [Worker node size resource reference](resource-reference-gcp.md#worker-node-size-resource-reference).

### Additional improvements 

- In the **Clusters** tab of the Cloud UI, you can now click a cluster entry to see details about the cluster configuration, including which **Worker Types** are enabled for the cluster.

> - You can now see the **Deployment ID** for each Deployment in the **Deployments** menu and on the Deployment information screen in the Cloud UI. A Deployment ID is required for deploying code using CI/CD.

Reviewer: ```suggestion
- The Deployment details page in the Cloud UI now includes an **ID** pane. The Deployment ID is required when you deploy code using the CI/CD process.
```

> - You can now access your Organization's **OpenLineage URL** from the **Settings** page of the Cloud UI.

Reviewer: ```suggestion
- An **OpenLineage URL** field is now available on the Cloud UI **Settings** page.
```

- Workspaces are now sorted alphabetically in the Cloud UI.

> - When you run `astro deploy` with an empty `dags` folder, Astro CLI version 1.8.0 or later excludes your `dags` folder when building and pushing an image of your project to Astro. This lets you manage your DAGs and project files in separate repositories when using [DAG-only deploys](deploy-code.md#deploy-dags-only).

Reviewer: ```suggestion
- In Astro CLI version 1.8.0 or later,  when you run `astro deploy` with an empty `dags` folder, the `dags` folder  is excluded when building and pushing an image of your project to Astro. This lets you manage your DAGs and project files in separate repositories when using [DAG-only deploys](deploy-code.md#deploy-dags-only).
```

### Bug fixes 

> - Fixed an issue where Astro temporarily stored DAGs for DAG-only deploys in a new directory called `/usr/local/airflow/dags/current`, which resulted in import errors in user code.

Reviewer: ```suggestion
- Fixed an issue where Astro temporarily stored DAGs for DAG-only deploys in a new directory named `/usr/local/airflow/dags/current`, which caused import errors in the user code.
```
